src/components/renderer/form-nested-screen.vue

```diff
@@ -54,21 +54,41 @@ export default {
     },
   },
   methods: {
+    elementsValid(config) {
+      config.forEach(item => {
+
+        //If the element has containers
+        if (Array.isArray(item)) {
+          this.elementsValid(item);
+        }
+
+        //If the element has items
+        if (item.items) {
+          this.elementsValid(item.items);
+        }
+
+        //hidden buttons
+        if (item.config && item.component === 'FormButton' && item.config.event === 'submit') {
```

Comment:
Instead of having a comment (which is confusing, since it's not explaining what this block is even doing, but simply stating "hidden buttons"), I think it would be much more readable to break out the conditional into a function. Something like:

```js
if (this.isSubmitButton(item)) {
  item.config.hidden = true;
}
```

Reply:
applied change